.travis.yml

```diff
@@ -1,22 +1,94 @@
 language: python
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
 sudo: false
-env:
-  - DJANGO_VERSION_CEILING=1.9 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.9 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.9 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.10 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
-  - DJANGO_VERSION_CEILING=1.11 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.11 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.11 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
-  - DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=sqlite
-  - DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
-  - DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
+
+matrix:
```

Comment:
Wow, this looks like a pain to maintain...
Unfortunately, since Django 2.0 only works with Python 3
and Django 1.x only works with Python 2, I don't think using `exclude` in the build matrix will be any shorter.

What about reving `django_richenum` to 4.0 and dropping support for Python 2 and Django 1.x?
We could keep a separate dev branch for `django_richenum` v3 for maintenance. I don't think it's worth trying to support Django 2 and Django 1.x in the same code base. We could also enable listing of the latest `django_richenum` v3 package version in PyPi to make the Python 2 people happy. See: https://wiki.python.org/moin/CheeseShopTutorial#Exposing_Multiple_Releases